grpc-gateway/pb/devices.proto

```diff
@@ -76,7 +77,8 @@ message SubscribeToEvents {
     }
     repeated Event event_filter = 1;
     repeated string device_id_filter = 2;
-    repeated string resource_id_filter = 3;
+    repeated string resource_id_filter = 3; // format {deviceID}{href}. eg "ae424c58-e517-4494-6de7-583536c48213/oic/d"
+    bool just_new_events = 4; // if it is set to true, the subscription sends only new events from creation time.
```

Comment:
This sounds strange in English.
After discussion with @oskarbauer, it is also strange to receive the current state in a first event if you are subscribing to WebSocket events. More clear would be to **not** receive current state in the first event by default and have a boolean `include_current_state`which if set to true (default false), first event containing current_state is sent.